lib/config/systemd_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package config
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/gravitational/teleport/lib/utils/golden"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSystemd_Install(t *testing.T) {
```

Comment:
nit: we're trying to get away from `_` in test names.
```suggestion
func TestWriteSystemdUnitFile(t *testing.T) {
```